lib/backend/syncersv1/updateprocessors/configurationprocessor_test.go

```diff
@@ -206,7 +207,7 @@ var _ = Describe("Test the generic configuration update processor and the concre
 			},
 		}
 		expected := map[string]interface{}{
-			"RouteRefreshInterval":      "12345",
```

Comment:
Is this the output that felix will see?  Sadly, felix has its own parse for these rather than parsing them as durations. If the felix value is in seconds, then it expects a float `10.5` with no `s` suffix, if the felix value is in millis, then it expects `10500.0`

Reply:
I was wondering about this, I'll see if I can get it to output floats from the converter.

Reply:
I also think it depends on the fields - some are millis and some are s.

@fasaxc :  Are all the duration/interval fields floats, or just the secs ones?  What about the millisecond interval fields?

Reply:
@robbrockbank They're all parsed as floats now.